docs/source/install.rst

```diff
@@ -1,265 +1,711 @@
 Installation
 ============
 
-The packages provided below correspond to branches hosted on GitHub.  The
-packages are updated through continuous integration via GitHub Actions for
-numerous Linux distributions.  RPM-based distributions are hosted on `openSUSE
-Build Service (OBS) <https://build.opensuse.org/>`_ while Debian-based
-distributions are hosted on `Launchpad <https://launchpad.net/>`_.
+This documentation covers how to use pre-built Linux OS packages to install
+GEOPM for all users of a system either using the latest stable release of GEOPM,
+or a development snapshot. The stable release packages are created as part of
+the Git repository tag/release process.  The development snapshot packages are
+built by GEOPM's GitHub CI process each time the ``dev`` branch is updated.
 
-Here are the packages curated by OBS and Launchpad:
 
-v3.1.0 to Current
------------------
+Install Latest Stable Release
+-----------------------------
 
-- ``geopm-service``: Sets up and enables the geopm systemd service.
-- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
-- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
-- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
-- ``geopm-doc``: Provides all GEOPM man pages.
+Install packages created by GEOPM development team to install the latest stable
+release of GEOPM.
 
-v3.0.1
-------
 
-- ``geopm-service``: Sets up and enables the geopm systemd service.
-- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
-- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
-- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
+GEOPM Access Service - Official
```

Comment:
Aren't all of our releases are equally official or unofficial? Maybe instead "stable" vs "development" packages?